dev_guide/service_accounts.adoc

```diff
@@ -170,6 +170,15 @@ To allow a secret to be mounted by a service account's pods, run:
 $ oc secrets link --for=mount <serviceaccount-name> <secret-name>
 ----
 
+[NOTE]
```

Comment:
@jim-minter I see what you mean, which is why I made sure the last part of the note box says "...mounting secrets to a service account's pods is not required." Or did you mean that it should be more obvious?